docs/graphqlserve/graphqlserve.md

```diff
@@ -107,6 +107,21 @@ Once we have a model with datasync capabilities, we can run our GraphQL server b
 ```bash
 gqlserve serve Note.graphql --datasync
 ```
+
+Conflict resolution strategies for datasync enabled models can be specified via the --conflict option:
+
+```bash
+gqlserve serve Note.graphql --datasync --conflict=client
```

Comment:
I think it is `clientSideWins` now instead of `client`

Reply:
Done this now. Thanks!